assets/src/components/nav.tsx

```diff
@@ -15,19 +16,26 @@ const Nav: React.FC<Props> = ({ children }) => {
   const [, dispatch] = useContext(StateDispatchContext)
   const deviceType = useScreenSize()
 
+  const { isViewOpen } = usePanelStateFromStateDispatchContext()
+
+  const navVisibilityStyle = isViewOpen ? "hidden" : "visible"
+
   switch (deviceType) {
     case "mobile":
       return (
         <div className="l-nav--narrow">
           <div className="l-nav__app-content">{children}</div>
-          <MobilePortraitNav />
+          <MobilePortraitNav isViewOpen={isViewOpen} />
         </div>
       )
     case "mobile_landscape_tablet_portrait":
       return (
         <div className="l-nav--medium">
           <div className="l-nav__app-content">{children}</div>
-          <div className="l-nav__nav-bar l-nav__nav-bar--left">
+          <div
+            className="l-nav__nav-bar l-nav__nav-bar--left"
+            style={{ visibility: navVisibilityStyle }}
```

Comment:
question: just wondering, why `visibility` here instead of something like the `hidden` attribute?

Reply:
Hmm, I feel like I'd need to know why that was chosen in `<MobilePortraitNav/>` originally. Using `hidden` instead of visibility would change the way it works(`visibility: visible` keeps the element taking up layout space, where `hidden` does `display: none`), and I'm not sure if there are assumptions else where that require that. But I would prefer `hidden` in both cases if there are no issues.

Reply:
It _seems_ like using `hidden`` would work? https://github.com/mbta/skate/pull/1719#discussion_r967366692

Reply:
I'll play around with it a bit and see if there are any drawbacks to `hidden`.